src/cider/exec/nextgen/context/CodegenContext.cpp

```diff
@@ -19,12 +19,25 @@
  * under the License.
  */
 #include "exec/nextgen/context/CodegenContext.h"
-
 #include "exec/nextgen/context/RuntimeContext.h"
 
 namespace cider::exec::nextgen::context {
 using namespace cider::jitlib;
 
+static std::unordered_map<SQLTypes, std::string> sql_type_name = {
+    {kNULLT, "null"},
+    {kTINYINT, "int8"},
+    {kSMALLINT, "int16"},
+    {kINT, "int32"},
+    {kBIGINT, "int64"},
+    // not supported
+    //  {SQLTypes::kFLOAT, "float"},
+    //  {SQLTypes::kDOUBLE, "double"},
+    //  {SQLTypes::kDATE, "date"},
+    //  {SQLTypes::kDECIMAL, "decimal"},
+    //  {SQLTypes::kBOOLEAN, "bool"},
+};
```

Comment:
the toString results in ```sqltypes.h``` are uppercase and don't contain size info, it will change the logic of runtime function like:
```C++
#define DEF_NEXTEGN_CIDER_SIMPLE_AGG_INT(width, aggname, aggfunc)         \
  extern "C" ALWAYS_INLINE void nextgen_cider_agg_##aggname##_int##width( \
      int##width##_t* agg_val_addr, const int##width##_t val) {           \
    aggfunc(*agg_val_addr, val);                                          \
  }
```

Reply:
Okay, I think you could implement a similar method in ```nextgen/utils/TypeUtils.h```, and label ```std::string toString(const SQLTypes& type)``` in ```sqltypes.h``` as deprecated.